py/runai_model_streamer/runai_model_streamer/file_streamer/tests/test_file_streamer.py

```diff
@@ -26,18 +27,19 @@ def test_runai_library(self):
             3: {"expected_text": "Test-Text3"},
         }
         with FileStreamer() as fs:
-            fs.stream_files([FileChunks(file_path, 1, request_sizes)])
-            for file, id, dst in fs.get_chunks():
-                self.assertEqual(file, file_path)
+            fs.stream_files([FileChunks(file_id, file_path, 1, request_sizes)], None, "cpu")
```

Comment:
In python, when you have multiple argument that have default value, you dont have to pass them to the function.

And if you do want to override only part of them, you can have it like that:
`fs.stream_files([FileChunks(file_id, file_path, 1, request_sizes)], ,device="cpu")`

Reply:
But if the default is "cpu" why do you need all these addition of `None, "cpu"`?